### PR TITLE
Firefox-114.0.1

### PR DESCRIPTION
**Pull request by n3m0-22**
https://www.mozilla.org/en-US/firefox/114.0.1/releasenotes/

### CODE REVIEW

**Review by mmstick — APPROVED** [review]

File: changelog [hunk 1]
```diff
@@ -0,0 +1,239 @@
+firefox (114.0.1) UNRELEASED; urgency=medium
```
Comment (13r0ck) [inline changelog:1]:
```suggestion
firefox (114.0.1) jammy; urgency=medium
```

**Review by 13r0ck — CHANGES_REQUESTED** [review]
Shasums match though!

**Review by 13r0ck — APPROVED** [review]
Thank you for being on top of these

**Review by leviport — DISMISSED** [review]
Isn't the changelog supposed to be in the /debian directory?

**Review by 13r0ck — APPROVED** [review]

**Review by jacobgkau — APPROVED** [review]
- [X] All listed bug fixes for the new release affecting Linux are working correctly
 - [X] `Settings` > `General` > `Firefox Updates` set to `Updates disabled by your system administrator`
 - [X] `Settings` > `Home` > `Firefox Home Content` > `Recommended by Pocket` unchecked by default
 - [X] Sync is working
 - [X] Videos services play (YouTube, Netflix, Amazon, Vimeo, etc)
    - [X] Sound and video
    - [X] Picture-in-picture 
    - [X] Subtitles in player, fullscreen, picture-in-picture
    - [X] Keyboard shortcuts in player, fullscreen, picture in picture
 - [X] Files save to Downloads by default
 - [X] Image can be set as background
    - [X] Dark mode
    - [X] Light mode

**Pull request merged**